examples/05_stable_diffusion/src/benchmark.py

```diff
@@ -282,6 +321,16 @@ def benchmark_diffusers(local_dir, batch_size, verify, benchmark_pt):
     np.random.seed(0)
     torch.manual_seed(4896)
 
+    """
+    Set the OS environment variable AITEMPLATE_WORK_DIR to point to an absolute
+    path to a directory which has AITemplate compiled artifacts the model(s). 
+    Make sure the OS user running this script has read and write permissions to 
+    this directory. By default, the it will look for compiled artifacts under 
+    tmp/ folder of the current working directory. 
+    
+    """
```

Comment:
Wondering whether it is necessary to have this repeated comment? Can it simply be moved into the source code of `get_work_dir_location_diffusers()`?

Reply:
Done